doc/en/user/source/community/jdbcstore/configuration.rst

```diff
@@ -0,0 +1,50 @@
+.. _community_jdbcstore_config:
+
+JDBCStore configuration
+========================
+
+The JDBCStore module is configured in the file ``jdbcstore/jdbcstore.properties`` inside the :ref:`datadir`.
+The following properties may be set:
+
+- ``enabled``: Use JDBCStore.  Turn off to use the data directory for all configuration instead.
+
+- ``initdb``: Initialize an empty database if this is set on true. 
+
+- ``import`` : The import configuration option tells GeoServer whether to import the current :ref:`datadir` from the file system to the database or not. If set to true, it will be imported and the config option will be set the value 'false' for the next start up to avoid trying to re-import the catalog  configuration.
+
+- ``initScript``: Path to initialisation script .sql file. Only used if initdb = true.
```

Comment:
Perhaps change "initdb = true" to "```initdb``` is true" for consistent monospace and preferring words over symbols in running text.